sdk/spring/tests.yml

```diff
@@ -12,9 +12,8 @@ stages:
           SubscriptionConfiguration: $(sub-config-gov-test-resources)
         China:
           SubscriptionConfiguration: $(sub-config-cn-test-resources)
-          Location: 'chinanorth'
+          Location: 'chinanorth3'
```

Comment:
Is this used for test? How about reverting unnecessary change?

Reply:
It's another story, 
it's `chinanorth3` in main branch, but changed to `chinanorth` when merging passwordless feature to main.